docs/source/app-dev/authorization.rst

```diff
@@ -198,4 +208,4 @@ The ``public`` right is implicitly granted to any request bearing a non-expired
 .. note:: All Daml ledgers also support a deprecated legacy format of custom Daml claims
    access tokens whose format is equal to the above except for the custom claims
    to be present at the same level as ``exp`` in the token above,
-   instead of being nested below ``"https://daml.com/ledger-api"``.
\ No newline at end of file
+   instead of being nested below ``"https://daml.com/ledger-api"``.
```

Comment:
Weird. What changed on this line?

Reply:
There was no newline in the end. My Intellij adds it automatically. Shall I remove it?

Reply:
Nah, thanks, I just couldn't figure out what changed. Thanks. 🙂